osh/client/completion/_osh-cli

```diff
@@ -76,6 +76,8 @@ _osh-cli_args()
                 '(-r --regex)'{-r,--regex}'[query by regular expression (python, module: re)]'
                 '(-p --pacakge)'{-p,--package}'[query by package name]'
                 '(-n --nvr)'{-n,--nvr}'[query by NVR (default one)]'
+                '(-s --state)'{-s,--state}'[query by task state. This option is used \
```

Comment:
Completions for ZSH:
```diff
diff --git a/osh/client/completion/_osh-cli b/osh/client/completion/_osh-cli
index 526f9af..8b77530 100644
--- a/osh/client/completion/_osh-cli
+++ b/osh/client/completion/_osh-cli
@@ -76,8 +76,7 @@ _osh-cli_args()
                 '(-r --regex)'{-r,--regex}'[query by regular expression (python, module: re)]'
                 '(-p --pacakge)'{-p,--package}'[query by package name]'
                 '(-n --nvr)'{-n,--nvr}'[query by NVR (default one)]'
-                '(-s --state)'{-s,--state}'[query by task state. This option is used \
-in conjunction with -r, -p, or -n.]'
+                '*'{-s+,--state=}'[query by task state. This option is used in conjunction with -r, -p, or -n.]:osh:->states'
             )
             ;;

@@ -172,6 +171,10 @@ in conjunction with -r, -p, or -n.]'
             _osh-cli_complete profiles
             ;;

+        states)
+            _osh-cli_complete task-states
+            ;;
+
         warn)
             local -a warn_levels
             warn_levels=(
```
It is rather tricky to write more complex completions for ZSH ao the explanation for the changes is the following:
* `*` for option means that repetition is allow, otherwise the option would be offered only once
* `(-s --state)` should not be present because it means that those two options conflict with each other.  So once `-s` was used, `--state` would not be offered.
* `-s+,--state=` means that the short and long options require an argument.
* `:osh:->states` defines the action to complete the option arguments.